src/components/sales/OrderBill.vue

```diff
@@ -386,7 +386,7 @@ export default defineComponent({
 	},
 	props:{
 		orderId: {
-			default: 0,
+			default: 1,
```

Comment:
1 is id number used by the backend. So make it defaults to zero and apply check on zero where necessary.